__tests__/unittest/templates.test.js

```diff
@@ -43,9 +43,7 @@ describe('templates', () => {
         it('should create API resource template correctly', () => {
             const serviceName = 'MyService';
             const srvDefinition = linkedModel
-            const packageIds = new Set()
-            packageIds.add('sap.test.cdsrc.sample:package:test-event:v1');
-            packageIds.add('sap.test.cdsrc.sample:package:test-api:v1');
+            const packageIds = ['sap.test.cdsrc.sample:package:test-event:v1', 'sap.test.cdsrc.sample:package:test-api:v1'];
```

Comment:
@zongqichen : Using `Array` instead of `Set` so we don't need to convert the `Set` into `Array` in different places, e.g., in the `_getPackageID` function.